opentech/apply/activity/messaging.py

```diff
@@ -379,6 +380,7 @@ class SlackAdapter(AdapterBase):
         MESSAGES.CREATED_PROJECT: '{user} has created a Project: <{link}|{source.title}>.',
         MESSAGES.UPDATE_PROJECT_LEAD: 'The lead of project <{link}|{source.title}> has been updated from {old_lead} to {source.lead} by {user}',
         MESSAGES.EDIT_REVIEW: '{user} has edited {review.author} review for <{link}|{source.title}>.',
+        MESSAGES.SEND_FOR_APPROVAL: '{user} has requested approval on project <{link}|{source.title}>.',
```

Comment:
:+1: but you haven't considered the recipients. In this case it'll be the Approvers. 

`def recipients` should be updated to return a list of slack ids for the Approvers.